pybaseball/split_stats.py

```diff
@@ -156,9 +156,9 @@ def get_splits(playerid: str, year: Optional[int] = None, player_info: bool = Fa
             pd.to_numeric, errors='coerce').convert_dtypes()
         level_data = level_data.dropna(axis=1, how='all')
         level_data = level_data.loc[playerid]
-        data = pd.concat([data, level_data])
+        # data = pd.concat([data, level_data])
     if player_info == False:
-        return data
+        return data, level_data if 'level_data' in locals() else None
     else:
         player_info_data = get_player_info(playerid=playerid, soup=soup)
-        return data, player_info_data
+        return data, player_info_data, level_data if 'level_data' in locals() else None
```

Comment:
Okay, can we do this without using symbol table introspection and that ternary there?
just
```python
if condition:
    return data, player_info_data
return data
```

Reply:
Updated.